includes/DefaultSettings.php

```diff
@@ -20,7 +20,7 @@
 $wgBaseMwEmbedPath = realpath( dirname( __FILE__ ) . '/../' );
 
 // The version of the library:
-$wgMwEmbedVersion = '2.74.1';
+$wgMwEmbedVersion = '2.74.3';
```

Comment:
We set version in release process with automated tools, please don’t change it manually.

Reply:
done. reverted to original value now that we are ready for deployment